docs/source/cloud-setup/policy.rst

```diff
@@ -0,0 +1,222 @@
+.. _advanced-policy-config:
+
+Admin Policy Enforcement
```

Comment:
Now that I think about it, "launch policy" is also appropriate, wdyt? Not asking for a rename, just an idea...

Reply:
Launch policy might be a bit inaccurate, since it is not only for launching but also for tasks execution.